sdk/avalon_sdk/work_order/work_order_params.py

```diff
@@ -179,23 +194,55 @@ def set_verifying_key(self, verifying_key):
     def add_in_data(self, data, data_hash=None,
                     encrypted_data_encryption_key=None, data_iv=None):
         """Add inData work order parameter."""
+        if data is None:
+            return util.create_error_response(
+                WorkOrderStatus.INVALID_PARAMETER_FORMAT_OR_VALUE,
+                0,
+                "Invalid data format for in data")
+
         in_data_copy = self.params_obj["inData"]
         new_data_list = self.__add_data_params(
             in_data_copy, data, data_hash, encrypted_data_encryption_key,
             data_iv)
 
         self.params_obj["inData"] = new_data_list
 
+        code, err = WOcheck.schema_validation(
```

Comment:
err  -> err_msg. Else it's confusing what is difference between code and err

Reply:
done